akka-stream/src/main/scala/akka/stream/javadsl/BidiFlow.scala

```diff
@@ -73,6 +75,17 @@ object BidiFlow {
   def fromFunctions[I1, O1, I2, O2](top: function.Function[I1, O1], bottom: function.Function[I2, O2]): BidiFlow[I1, O1, I2, O2, Unit] =
     new BidiFlow(scaladsl.BidiFlow.fromFunctions(top.apply _, bottom.apply _))
 
+  /**
+   * If the time between two processed elements *in any direction* exceed the provided timeout, the stream is failed
+   * with a [[java.util.concurrent.TimeoutException]].
+   *
+   * There is a difference between this stage and having two idleTimeout Flows assembled into a BidiStage.
+   * If the timeout is configured to be 1 seconds, then this stage will not fail even though there are elements flowing
+   * every second in one direction, but no elements are flowing in the other direction. I.e. this stage considers
+   * the *joint* frequencies of the elements in both directions.
+   */
+  def idleTimeout[I, O](timeout: FiniteDuration): BidiFlow[I, I, O, O, Unit] =
```

Comment:
it has the same name as the FlowOps version but different semantics than two of those naively put into a BidiStage. I just want to avoid confusion because of misinterpreted semantics.